src/components/newsItem.js

```diff
@@ -10,6 +10,32 @@ export default function NewsItem({
   id,
   hoverEffects,
 }) {
+  const renderWrapperContent = () => {
+    return (
+      <React.Fragment>
+        <div className="news-item__back-panel"></div>
+        <div className="news-item__wrapper">
+          <div className="news-item__image-container">
+            {image && (
```

Comment:
Since you are already doing checks here for image, title, etc. I think you should be safe to remove the reducer from the archive.

Typically what I like to do is leave as much control to the content creator as possible to enter whatever content they want. If something is absolutely required, then we can set it up to be a required field in Contentful. Otherwise, we just check for what they have given and then make sure it displays nicely for every combination of data sent through.

Reply:
I'll update this so that news items without all of the data are still displayed. I do need the reducer in the news archive to get things organized. Let me know if you think it would be better to organize the data directly in a news item and I can move it over!